node/src/Pos/Wallet/Web/Pending/Types.hs

```diff
@@ -89,6 +90,9 @@ instance Buildable PtxCondition where
         PtxWontApply reason _ ->
             bprint ("wont apply ("%build%")") reason
 
+instance Buildable (SecureLog PtxCondition) where
+    build = buildUnsecure
```

Comment:
I have no idea what's going on, but it looks suspicious that `Buildable` instance for `SecureLog whatever` uses `unsecure`.

Reply:
Fixed